### PR TITLE
Dynamic security analysis documentation

### PR DESCRIPTION
**Pull request by thangqp**
Add also documentation for time-series server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,16 +241,38 @@ This service is responsible for running a short-circuit analysis and storing res
...
 
 - Kind: Web service with a REST API
 - Source repository: https://github.com/gridsuite/dynamic-simulation-server
 - Storage: PostgreSQL
 - Connected to message broker: producer and consumer
-- Other services dependencies: network store server
+- Other services dependencies: network store server, report server, dynamic mapping server, time-series-server
+- Use PowSyBl libraries: yes
+
+This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet completely integrated into GridStudy.
```
Comment (etiennehomer) [inline README.md:264]:
```suggestion
This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). 
```

Comment (etiennehomer) [reply README.md:264]: This documentation is not updated enough to keep some "it is ongoing work...". Let's make it simple so we don't need to keep it up to date later on this point. 
And for the opensource point of view, dynamic simulation is integrated

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -241,16 +241,38 @@ This service is responsible for running a short-circuit analysis and storing res
...
+
+- Kind: Web service with a REST API
+- Source repository: https://github.com/gridsuite/dynamic-security-analysis-server
+- Storage: PostgreSQL
+- Connected to message broker: producer and consumer
+- Other services dependencies: network store server, report server, dynamic simulation server
 - Use PowSyBl libraries: yes
 
-This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet integrated into GridStudy.
+This service is responsible for running dynamic security analysis. It is based on [PowSyBl dynamic security analysis API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-security-analysis) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet completely integrated into GridStudy.
```
Comment (etiennehomer) [inline README.md:275]: Same here

File: README.md [hunk 3]
```diff
@@ -261,7 +283,7 @@ This service is responsible for running dynamic simulations. It is based on [Pow
 - Other services dependencies: network store server
 - Use PowSyBl libraries: yes
 
-This service is responsible for configuring how dynamic data is mapped to a network. This is still an ongoing work and is not yet integrated with dynamic simulation server.
+This service is responsible for configuring how dynamic data is mapped to a network. This is still an ongoing work and is not yet completely integrated with dynamic simulation server.
```
Comment (etiennehomer) [inline README.md:286]: same

**Review by etiennehomer — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (thangqp) [reply README.md]: DONE in https://github.com/gridsuite/documentation/pull/16/commits/c536a6b68417567e27b8c6840abbf5c4a42c09dd

File: README.md (hunk 3 above)
Comment (thangqp) [reply README.md]: DONE in https://github.com/gridsuite/documentation/pull/16/commits/c536a6b68417567e27b8c6840abbf5c4a42c09dd

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**